website/docs/api/spancategorizer.mdx

```diff
@@ -20,8 +20,9 @@ output class probabilities are independent for each class. However, if you need
 to predict at most one true class for a span, then use `spancat_singlelabel`. It
 uses a `Softmax` layer and treats the task as a multi-class problem.
 
-Predicted spans will be saved in a [`SpanGroup`](/api/spangroup) on the doc.
-Individual span scores can be found in `spangroup.attrs["scores"]`.
+Predicted spans will be saved in a [`SpanGroup`](/api/spangroup) on the `doc.spans["spans_key"]`,
+where `"spans_key"` can be specified for the component.
```

Comment:
Something more like this (plus formatting):

```suggestion
Predicted spans will be saved in a [`SpanGroup`](/api/spangroup) on the doc under `doc.spans[spans_key]`,
where `spans_key` is a component config setting.
```

(In other places we have `Doc.spans[spans_key]`, maybe this should be made consistent, too.)